libs/application/templates/children-residence-change/src/forms/ChildrenResidenceChangeForm.ts

```diff
@@ -86,6 +86,12 @@ export const ChildrenResidenceChangeForm: Form = buildForm({
                   title: m.externalData.otherParents.title,
                   subTitle: m.externalData.otherParents.title,
                 }),
+                buildDataProviderItem({
+                  id: 'userProfile',
+                  type: 'UserProfileProvider',
+                  title: '',
+                  subTitle: '',
```

Comment:
I am deliberately leaving this empty since I would argue that just logging in gives the consent of accessing your `email` and `phoneNumer` - If not then we will add in a title and a subtitle

Reply:
Agree we should though let Baldur and Frilli know about this and see if we are all on the same page.